### ISSUE TITLE
Can't open Players menu

### ISSUE THREAD

**Issue by silkdevelopment** [Project-Entity/pe-adminmenu#10]
![123](https://user-images.githubusercontent.com/81208494/112091536-8c67bf80-8bd0-11eb-951c-63bc43f37010.png)
admin_cl.lua:267: attempt to concatenate a nil value (field 'name')

**Comment by XiosBombay** [comment]
What esx version are you using?

**Comment by silkdevelopment** [comment]
> What esx version are you using?

I'm using esx version 1.1 (the old one, using essentialmode)

**Comment by XiosBombay** [comment]
I'm not used to ESX 1.2. So you could trying this [line](https://github.com/Project-Entity/pe-adminmenu/blob/master/server/admin_sv.lua#L51) to xPlayer.getName()

**Comment by silkdevelopment** [comment]
Still I got the error :( What should I do? This is the only error I got, the rest your script is awesome.

**Comment by XiosBombay** [comment]
You have esx_identity right? If it doesn't work it's either because xPlayer.getName() doesn't exist or because the players don't have a character name. Either way. This will most likely be re-worked on the following updates.

**Comment by silkdevelopment** [comment]
Yeah, i'm using esx_identity. I don't know what to put for that line. Yes please, looking forward to fix this issue. Thank you!

**Comment by silkdevelopment** [comment]
Hi sir, do you have fix on this issue? thank you so much

**Comment by XiosBombay** [comment]
It has been fixed V2. The entire system is not done yet but it's in progress.
